### PR TITLE
Add more use cases and remove internal details of some use cases

### PR DESCRIPTION
**Pull request by tyanhan**

### CODE REVIEW

**Comment by tyanhan** [comment]
Also added ImportSchedule manual testing and standardised indentation for use cases

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -631,80 +631,95 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-  Use case ends.
+    Use case ends.
 
 * 2a. The file information is in the wrong format.
 
-    * 2a1. UniGenda shows an error message.
-    * 2a2. If app can save file, UniGenda creates an example file for user reference.
-    * 2a3. If app cannot save the file, UniGenda shows an example in the display box.
+  * 2a1. UniGenda shows an error message.
+  * 2a2. If app can save file, UniGenda creates a template file for user reference.
```
Comment (hanqinilnix) [inline docs/DeveloperGuide.md:719]: Does this change really matter?

File: docs/team/tyanhan.md [hunk 2]
```diff
@@ -14,29 +14,39 @@ Given below are my contributions to the project.
...
+
 * **Enhancements to existing features**:
   * Fixed the GUI size and removed the resizable icon on mouse hover bug. [#121](https://github.com/AY2122S2-CS2103T-W09-1/tp/pull/121)
   * Wrote tests for `AddEventCommand`, `ClearScheduleCommand`, `SetUserCommand` and their related parsers. [#84](https://github.com/AY2122S2-CS2103T-W09-1/tp/pull/84/files) [#119](https://github.com/AY2122S2-CS2103T-W09-1/tp/pull/119) [#113](https://github.com/AY2122S2-CS2103T-W09-1/tp/pull/113)
   * Wrote tests for `JsonAdaptedEvent` and `JsonAdaptedSchedule` [#77](https://github.com/AY2122S2-CS2103T-W09-1/tp/pull/77/files)
+
 * **Documentation**:
   * User Guide:
     * Added documentation for the `AddEvent`, `ImportSchedule`, `ClearSchedule` and `SetUser` features.
+    * Wrote a brief description of the product and organised the command summary.
```
Comment (hanqinilnix) [inline docs/team/tyanhan.md:49]:
```suggestion
    * Refine description of the product and organised the command summary.
```

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (tyanhan) [reply docs/DeveloperGuide.md:719]: I think it is clearer.

**Comment by vishandi** [comment]
LGTM

**Pull request merged**